.github/workflows/windows.yml

```diff
@@ -1,31 +0,0 @@
-# http://webui.me
-# https://github.com/webui-dev/zig-webui
-# Licensed under MIT License.
-# All rights reserved.
-
-name: Windows
-on:
-  workflow_call:
-
-jobs:
-  build_nightly:
-    runs-on: windows-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: goto-bus-stop/setup-zig@v2
-      - name: build_examples_static
-        run: zig build build_all
-      # - name: build_examples_dynamic
-      #   run: zig build build_all -Dis_static=false
```

Comment:
Right it's just the dynamic builds that are commented out. Was hard to check on the phone. You are right to enable them

Reply:
Gonna update in a sec